base_decorator.rb

```diff
@@ -0,0 +1,14 @@
+require './nameable'
```

Comment:
- [ ] In your readme file, kindly add the tech stack used to build this project (Ruby). This will make your repo more professional.
![image](https://user-images.githubusercontent.com/94551484/223485949-acd6d417-3ce1-4a29-b9f4-8d8d6cf78ad5.png)

- [ ] In your readme file, kindly add some features you will add under the future features section. This will make your repo more professional.
![image](https://user-images.githubusercontent.com/94551484/223486231-d0a4b96c-4653-457a-8acf-9f9406c89616.png)